org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/formatter/BaseFormatter.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.ls.debug.adapter.formatter;
+
+import com.sun.jdi.Type;
+import com.sun.jdi.Value;
+
+public abstract class BaseFormatter implements IValueFormatter {
```

Comment:
minor, how about BaseValueFormatter considering you also have a TypeFormatter kind of thing?